atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -102,20 +104,29 @@ static void waitForSchemaVersions(
 
         StringBuilder schemaVersions = new StringBuilder();
         for (Entry<String, List<String>> version : versions.entrySet()) {
-            schemaVersions.append(String.format("%nAt schema version %s:", version.getKey()));
-            for (String node : version.getValue()) {
-                schemaVersions.append(String.format("%n\tNode: %s", node));
-            }
+            schemaVersions = addNodeInformation(schemaVersions,
+                    String.format("%nAt schema version %s:", version.getKey()),
+                    version.getValue());
         }
+
+        String configNodes = addNodeInformation(new StringBuilder(),
+                "Nodes specified in config file:",
+                config.servers().stream().map(InetSocketAddress::getHostName).collect(Collectors.toList()))
+                .toString();
+
         String errorMessage = String.format("Cassandra cluster cannot come to agreement on schema versions,"
                         + " after attempting to modify table %s. %s"
-                        + " \nFind the nodes above that diverge from the majority schema"
-                        + " or have schema 'UNKNOWN', which likely means they are down/unresponsive"
+                        + " \nFind the nodes above that diverge from the majority schema. If nodes"
+                        + " have schema 'UNKNOWN', they are likely down/unresponsive"
                         + " and examine their logs to determine the issue."
                         + " Fixing the underlying issue and restarting Cassandra should resolve the problem."
-                        + " You can quick-check this with 'nodetool describecluster'.",
+                        + " You can quick-check this with 'nodetool describecluster'."
+                        + " \nIf nodes are specified in the config file, but do not have a schema version,"
+                        + " then they have never joined the cluster and you should verify your configuration"
+                        + " is correct. %s",
```

Comment:
Let's change "and you should verify your configuration is correct" to "and you should verify that the nodes specified in the config are up and have joined the Cassandra cluster. Is this what they should do in this case? Should they contact support?